### PR TITLE
Add json content validation to presubmit

### PR DESCRIPTION
**Pull request by keyonghan**
Recently supported LUCI try builders configuration has been available in cocoon repo in a JSON file. This PR validates the json format in presubmit tasks.

### CODE REVIEW

File: dev/validate_json.dart [hunk 1]
```diff
@@ -3,22 +3,30 @@
...
-Future<bool> main(List<String> args) async {
+Future<void> main(List<String> args) async {
   final String jsonString = await File(args[0]).readAsString();
   Map<String, dynamic> decodedJson;
-  bool decodedStatus = true;
+  final List<String> messages = <String>[];
   try {
     decodedJson = json.decode(jsonString) as Map<String, dynamic>;
     final List<dynamic> builders = decodedJson['builders'] as List<dynamic>;
     if (builders == null) {
```
Comment (digiter) [inline dev/validate_json.dart:27]: The current implementation doesn't check the schema of a builder.

A better way is adding a @JsonSerializable() class representing the schema of the json, then parse the json with that class. We need to parse the json in Cocoon backend code anyway.

Comment (keyonghan) [reply dev/validate_json.dart:29]: Good point! Have added logic to check builder schema.

File: dev/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: dev_check
+description: Script to check licenses.
```
Comment (godofredoc) [inline dev/pubspec.yaml:2]: Update description?

File: dev/pubspec.lock [hunk 3]
```diff
@@ -0,0 +1,369 @@
+# Generated by pub
```
Comment (godofredoc) [inline dev/pubspec.lock:1]: Do we really need all these packages to validate a json file or we need an updated file?

File: dev/pubspec.yaml (hunk 2 above)
Comment (keyonghan) [reply dev/pubspec.yaml]: updated.

File: dev/pubspec.lock (hunk 3 above)
Comment (keyonghan) [reply dev/pubspec.lock:1]: removed unnecessary dependencies, and re-ran `pub get`. Not much change though.

**Pull request merged**